spoon-runner/src/main/java/com/squareup/spoon/SpoonRunner.java

```diff
@@ -322,6 +329,11 @@ public Builder setMethodName(String methodName) {
       return this;
     }
 
+    public Builder setTestRunListeners(List<ITestRunListener> testRunListeners) {
```

Comment:
This should take a single `ITestRunListener` and add it to the list (which is initialized to an empty `ArrayList`).